### PR TITLE
(fix) debugging backspace issues with hr

### PR DESCRIPTION
**Pull request by dselman**
Signed-off-by: Dan Selman <danscode@selman.org>

# Issue #<NUMBER HERE>
<OVERALL SUMMARY OF PULL REQUEST>

### Changes
- <INSERT DESCRIPTION OF CHANGES>
  - <SUB NOTES OF DESCRIPTIONS>
- <SUMMARIZE ALL COMMITS IN PULL REQUEST>

### Flags
- <DESCRIBE ISSUES OR HOLDS FOR REVIEWERS>

### Related Issues
- Issue #<NUMBER HERE>
- Pull Request #<NUMBER HERE>

### CODE REVIEW

**Comment by jeromesimeon** [comment]
I was testing by deleting from the end of the document. The delete skipped over the thematic break.
I think it worked if I delete from the previous paragraph.

**Comment by dselman** [comment]
> I was testing by deleting from the end of the document. The delete skipped over the thematic break.
> I think it worked if I delete from the previous paragraph.

This seems to work for me in Safari and Chrome.
https://deploy-preview-233--accordproject-markdown-editor.netlify.com/examples/

**Comment by jeromesimeon** [comment]
Yes, I think it's only when the thematic break is the last node in the document? it's a bit of an edge case.

BackspaceThematicBreak

**Comment by dselman** [comment]
> Yes, I think it's only when the thematic break is the last node in the document? it's a bit of an edge case.
> 
> BackspaceThematicBreak

Interesting. I can reproduce that on Netlify, but not locally.

**Comment by dselman** [comment]
I can also reproduce this with images. It seems to be an issue deleting void (?) blocks.

**Comment by jeromesimeon** [comment]
Could this be in the logic that attempts to add an empty text node at the end?

File: src/styles.css [hunk 1]
```diff
@@ -13,6 +13,15 @@ body {
     text-indent: 0ex;
 }
 
+.hr {
+    overflow: visible; /* For IE */
+    padding: 0;
+    border: none;
+    border-top: medium double #333;
```
Comment (DianaLease) [inline src/styles.css]: why double? do we want an `hr` to appear as two lines?

Comment (dselman) [reply src/styles.css]: I've no opinion on that... Whatever people prefer. The bug seems to be fixed moving from an `<hr>` to a `<div>` --- not really sure *why* and we have the same issue with the `<image/>` tag.

**Pull request merged**